src/app.py

```diff
@@ -168,6 +168,12 @@ def new_class():
     except UrlNameAlreadyTakenError as e:
         return jsonify({"error": "Url name was already taken!", "exception": str(e)}), 403
 
+@app.route('/classes/<class_url>/update_info/')
+def update_class_description(class_url):
+    dbw = DatabaseWrapper(GITOLITE_ADMIN_PATH, DATABASE_PORT)
+    new_description = request.form.get("description")
```

Comment:
Want to be able to also update long_name and owner. If any of these come in as `None` we should keep the original values